modules/home.nix

```diff
@@ -0,0 +1,12 @@
+#TODO make this more of a home.nix setup?
+{config, pkgs, ...}: 
+{
+    home.username ="ezekiel";
+    home.homeDirectory = "/home/ezekiel";
+    programs.home-manager.enable =true;
+    home.stateVersion = "23.11";
+
+    home.package = [
```

Comment:
`home-manager.users.ezekiel.home.package` does not exist.
Did you mean `home-manager.users.ezekiel.home.packages`?

Reply:
oh yeah totally, main thing is am i on the right track?
so i got a home.nix, and then configs for each package, my biggest concern is whether or not 
the i3.nix is gonna work.... especially since i wanna also config things like kitty